pkg/entity/datapolicy/datapolicy.go

```diff
@@ -139,8 +140,16 @@ func getBlueprintPolicyFromProcessingPlatform(platformId string, tableId *string
 	}
 }
 
-func list(_ *cobra.Command) {
-	req := &ListDataPoliciesRequest{}
+func list(cmd *cobra.Command) {
+	flags := cmd.Flags()
```

Comment:
Maybe you can create a function that creates the `PageParameters` struct from the flags?